test/integration/multinode_test.go

```diff
@@ -62,7 +62,7 @@ func TestMultiNode(t *testing.T) {
 
 func validateMultiNodeStart(ctx context.Context, t *testing.T, profile string) {
 	// Start a 2 node cluster with the --nodes param
-	startArgs := append([]string{"start", "-p", profile, "--wait=true", "--memory=2200", "--nodes=2"}, StartArgs()...)
+	startArgs := append([]string{"start", "-p", profile, "--wait=true", "--memory=2200", "--nodes=2", "--alsologtostderr"}, StartArgs()...)
```

Comment:
lets add -v=8 to get the libmachine things too

Reply:
libmachine is enabled automatically for --alsologtostderr